public/src/components/channelManagement/headTests/headTestEditor.tsx

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import { Region } from '../../../utils/models';
+
+import { UserCohort } from '../helpers/shared';
+
+import { Typography } from '@material-ui/core';
+import HeadTestVariantEditor from './headTestVariantEditor';
+import TestVariantsEditor from '../testVariantsEditor';
+import TestEditorHeader from '../testEditorHeader';
+
+import TestEditorTargetAudienceSelector from '../testEditorTargetAudienceSelector';
+
+import TestEditorActionButtons from '../testEditorActionButtons';
+import LiveSwitch from '../../shared/liveSwitch';
+import useValidation from '../hooks/useValidation';
+import { HeadTest, HeadVariant } from '../../../models/head';
+import { getDefaultVariant } from './utils/defaults';
+import TestEditorVariantSummary from '../testEditorVariantSummary';
+
+import { ControlProportionSettings } from '../helpers/controlProportionSettings';
+import TestVariantsSplitEditor from '../testVariantsSplitEditor';
+import { useStyles } from '../helpers/testEditorStyles';
+
+interface HeadTestEditorProps {
+  test: HeadTest;
+  hasChanged: boolean;
+  onChange: (updatedTest: HeadTest) => void;
+  onValidationChange: (isValid: boolean) => void;
+  visible: boolean;
+  editMode: boolean;
+  onDelete: () => void;
+  onArchive: () => void;
+  onTestSelected: (testName: string) => void;
+  testNames: string[];
+  testNicknames: string[];
+  createTest: (newTest: HeadTest) => void;
+}
+
+const HeadTestEditor: React.FC<HeadTestEditorProps> = ({
```

Comment:
why `HeadTest` on the client? It's inconsistent with the scala and SDC, where it's `HeaderTest`

Reply:
Happy to revert to `Header...` (and `header...`) across all files (including filenames)